dev/ledger.md

```diff
@@ -523,6 +535,100 @@ valid vote round in its participation key is strictly less than the current roun
+   under msgpack key `v`. 
+
+ - The $\ln(ProvenWeight)$ with 16 bits of precision that would used to verify state proof for rounds ((_X_+1)$\cdot$$\delta_{SP}$,...,(_X_+2)$\cdot$$\delta_{SP}$], under msgpack key `P`. 
+
+# State Proof Tracking
+
+Each block header keeps track of the state needed to construct, validate,
+and record state proofs.  
+This tracking data is stored in a map under the msgpack key `spt` in the block header.
+The map is indexed by the type of the state proof; at the moment, only
+type 0 is supported.  In the future, other types of state proofs
+might be added.
+
+For type 0, $KQ_{SP}=256$, $f_{SP}$ is $2^{32}*30/100$
+(as the numerator of a fraction out of $2^{32}$), $N_{SP}=1024$,
+$\delta_{SP}=256$, $\delta_{SPR}=10$  and $\delta_{SPB}=16$ .
+
+The value of the tracking data is a msgpack map with three
+elements:
+
+- Under key `n`, the next expected round of a state proof that
+  should be formed.  When upgrading from an earlier consensus protocol
+  to a protocol that supports state proofs, the `n` field is
+  set to the lowest value such that `n` is a multiple of $\delta_{SP}$
+  and so that the `n` is at least the first round of the new protocol
+  (supporting state proofs) plus $\delta_{SPB}+\delta_{SP}$.
+  This field is set in every block.
+
+- Under key `v`, the root of the Vector commitment to an array of
+  participants that are eligible to vote in the state proof at
+  round $\delta_{SP}$ from the current block.  Only blocks whose round
+  number is a multiple of $\delta_{SP}$ have a non-zero `v` field.
+
+- Under key `t`, the total weight of participants in the Vector commitment.
+
+The participants committed to by the Vector commitment are chosen in a
+specific fashion:
+
+- First off, because it takes some time to collect all of the online
+  participants (more than the target assembly time for a block), the
+  set of participants appearing in a commitment in block at round $r$
+  are actually based on the account state from round $r-\delta_{SPB}$.
+
+- The participants are sorted by the number of microAlgos they currently
+  hold (including any pending rewards).  This enables more compact
+  proofs of pseudorandomly-chosen participants weighted by their
+  microAlgo holdings.  Only accounts in the online state are included
+  in this list of participants.
+
+- To limit the worst-case size of this Vector commitment, the array of
+  participants contains just the top $N_{SP}$ participants.  Efficiently
+  computing the top $N_{SP}$ accounts by their algo balance is difficult
+  in the presence of pending rewards.  Thus, to make this top-$N_{SP}$
+  calculation more efficient, we choose the top accounts based on a
+  normalized balance.  The normalized balance is a hypothetical balance
+  that a given account would need to have at round 0 to achieve its
+  current balance (without pending rewards) as of the last round at
```

Comment:
```suggestion
  balance (without pending rewards) as of the last round at
```
Is it the "current" balance or the one as of the last round at which the account was toucher?